src/tests/mod.rs

```diff
@@ -0,0 +1,11 @@
+#![allow(clippy::cognitive_complexity)]
```

Comment:
Since this is in the `src` directory now, which has this attribute in `lib.rs`, we can remove it from here

Reply:
fixed